docs/utilities/tags.md

```diff
@@ -5,24 +5,24 @@ Tags are generalized sets of objects in the game used for grouping related thing
 
 Declaring Your Own Groupings
 ----------------------------
-Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid>/tags/blocks/foo/tagname.json` will declare a `Tag<Block>` with ID `modid:foo/tagname`.
+Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid>/tags/blocks/foo/tagname.json` will declare an `INamedTag<Block>` with ID `modid:foo/tagname`.
 Similarly, you may append to or override tags declared in other domains, such as Vanilla, by declaring your own JSONs.
 For example, to add your own mod's saplings to the Vanilla sapling tag, you would specify it in `/data/minecraft/tags/blocks/saplings.json`, and Vanilla will merge everything into one tag at reload, if the `replace` option is false.
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
 See the [Vanilla wiki][tags] for a description of the base syntax.
 
 Forge provides two extensions on the Vanilla syntax:
 * You may declare an `optional` array of the same format as the `values` array, but any values listed here that are not present will not cause the tag loading to error.
-This is useful to provide integration for mods that may or may not be present at runtime.
+This has been deprecated in favor of the vanilla method of specifying optional tag values.
```

Comment:
Is it worth mentioning the vanilla method for this? `required: false` iirc

Reply:
I was considering it, but we do already direct the user to view the vanilla syntax in the wiki. It would appropriate to create more examples, but I also felt that was a tiny bit out of scope for the omnibus.